packages/applications-service-api/src/routes/documents.v3.js

```diff
@@ -2,9 +2,22 @@ const express = require('express');
 
 const documentsV3Controller = require('../controllers/documents.v3');
 const { validateRequestWithOpenAPI } = require('../middleware/validator/openapi');
+const config = require('../lib/config');
+const { asyncRoute } = require('@pins/common/src/utils/async-route');
 
 const router = express.Router();
 
-router.post('', validateRequestWithOpenAPI, documentsV3Controller.getDocuments);
+const getDocumentsRoute = (req, res, next) => {
+	const backOfficeCaseReferences =
+		config.backOfficeIntegration.documents.getDocuments.caseReferences || [];
+
+	const route = backOfficeCaseReferences.includes(req.body.caseReference)
```

Comment:
In the future we might want to extend this to have a global switch to BO, probably best on another ticket to handle all routes.

Reply:
Yes, probably one to cover off in another ticket when we have a better idea how we want to go live with a BO/NI hybrid solution